ConvertAssets.py

```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/python
```

Comment:
This file doesn't hurt anyone by being there, if people prefer having their python script to convert CamelCase to snake_case, then so be it.
It does not add any setup time or anything for other people.